Cargo.toml

```diff
@@ -1,27 +1,29 @@
 [package]
 name = "libflate"
-version = "1.4.0"
+version = "2.0"
```

Comment:
```suggestion
version = "2.0.0"
```

Also, why the major version bump? Does this PR break any APIs?

Reply:
IMO, removing an existing feature (`no_std` in this case) is a breaking change. But, feel free to comment if you think it's unnecessary.

Reply:
I'll consider this point in more detail later. As @kupiakos said, the `no_std` feature approach in the master branch has some problems. So, it might be better to treat this PR as a bug fix not a breaking change.

Reply:
I don't have a preference either way, just curious.

Reply:
It's a breaking change that's particularly hard for users to migrate if needed; I'll leave as `2.0.0`

Reply:
Done.